CHANGELOG.md

```diff
@@ -9,6 +11,7 @@ MapWidget(
   ),
 );
 ```
+* Update Turf dependency to `4.0.0-beta.1`
```

Comment:
It's an internal change on iOS, does it affect Dart side to be mentioned here?

Reply:
I included a note as an FYI as I ran into some build issues. They were resolved, but I thought it would be better to mention.